### DISCUSSION TITLE
Using theme "90210" how to set show full path

### DISCUSSION THREAD

**Discussion by khailifz** [ohmyzsh/ohmyzsh#12526] [Help]
If I go into a very deep directory, this theme will not show the full current path. How can I set it up?

**Comment by mcornella** [comment]
Looks like you might be referring to [this Oh My Bash theme](https://github.com/ohmybash/oh-my-bash/blob/master/themes/90210/90210.theme.sh)? If so, you should try asking on https://github.com/ohmybash/oh-my-bash.
